server/src/routes/internship.ts

```diff
@@ -87,29 +105,18 @@ internshipRouter.patch(
   "/:id",
   authMiddleware(),
   param("id").custom(isObjectId),
-  query([
-    "startDate",
-    "endDate",
-    "tasks",
-    "operationalArea",
-    "livingCosts",
-    "salary",
-    "workingHoursPerWeek",
-    "programmingLanguages", //array
-    "paymentTypes", //enum, array
-    //supervisor
-    "supervisorFullName",
-    "supervisorEmailAddress",
-  ]),
+  query(standardPostParams),
   validate,
   asyncHandler(updateInternship)
 );
 
 /* PDF endpoints */
+const idRegEx = /[0-91-f]{24}/;
```

Comment:
```suggestion
const idRegEx = /[0-9a-f]{24}/;
```
Ich glaube, dass sollte eigentlich ne Hex-Regex sein 😅